core/vmpu/src/kinetis/vmpu_kinetis_mem.c

```diff
@@ -63,20 +63,30 @@ void vmpu_mem_switch(uint8_t src_box, uint8_t dst_box)
 
 void vmpu_mem_init(void)
 {
+    /* UART DMA is bus master 2 (configured through M2UM). ENET DMA is bus
+     * master 3 (configured through M3UM). */
+    static const uint32_t bmum_read_bit = (1 << 2);
+    static const uint32_t bmum_write_bit = (1 << 1);
+#define M3UM 18
```

Comment:
I'd rather not use hardcoded hex numbers, but names from the datasheet (easy to search, easy to know what one is looking at).

M3UM is useful in that it matches the register name in the data sheet. Also, there isn't a need to add all defines for all these registers, just the ones we need.

I'm not sure what style we like better regarding defines or static const. Where possible, I prefer static const.

I'll revise this a bit and you can see if you like it better.